responsibleai/responsibleai/managers/error_analysis_manager.py

```diff
@@ -111,6 +124,30 @@ def _apply_func(self, func, dataset):
         return func(dataset.drop(columns=self.dropped_features, axis=1))
 
 
+class MetadataRemovalRegressionModelWrapper():
```

Comment:
An alternative might have also been to create a base class with the predict method and have classification/regression extend it, but I'm not sure if that would have saved enough code lines anyway, although maybe it might have reduced a few duplicate lines to a common place.  I think this might still be better.

Reply:
thanks for the great comment!
Since we want to merge this PR today, I will merge it now as all tests passed. And I will address this comment in my next PR.